paasta_tools/cli/cmds/local_run.py

```diff
@@ -923,8 +924,15 @@ def configure_and_run_docker_container(
     )
 
 
+def docker_config_available():
+    home = Path.home()
+    oldconfig = os.path.join(home, ".dockercfg")
+    newconfig = os.path.join(home, ".docker", "config.json")
+    return os.path.isfile(oldconfig) or os.path.isfile(newconfig)
```

Comment:
To be safe, it might be better to check if the file is readable rather than whether it merely exists.

Reply:
Got it